config.py

```diff
@@ -1,26 +1,32 @@
 import os
-import json
 import time
-import inspect
 import shutil
 from pathlib import Path
+
 from filelock import FileLock
-from types import SimpleNamespace
-from dataclasses import dataclass, field
 from abc import ABC, abstractmethod
-from typing import Callable, Any, MutableSequence
-
-
-@dataclass
-class AlltalkConfigTheme:
+from typing import Callable, MutableSequence
+
+from pydantic import BaseModel, ConfigDict, AliasGenerator, AliasChoices, Field
+
+class AlltalkConfigTheme(BaseModel):
+    # Map 'class' to 'clazz' and vice versa
+    model_config = ConfigDict(

```

Comment:
That's how pydantic handles mapping of fields, in this case for `clazz` to `class`.